src/server/protocol.ts

```diff
@@ -546,7 +546,15 @@ namespace ts.server.protocol {
         command: CommandTypes.GetApplicableRefactors;
         arguments: GetApplicableRefactorsRequestArgs;
     }
-    export type GetApplicableRefactorsRequestArgs = FileLocationOrRangeRequestArgs;
+    export type GetApplicableRefactorsRequestArgs = FileLocationOrRangeRequestArgs & {
+        triggerReason?: RefactorTriggerReason;
+    };
+
+    export type RefactorTriggerReason = RefactorInvokedReason;

```

Comment:
Why re-export it under an additional name?

Reply:
I think this was from following the pattern for `SignatureHelpTriggerReason`. I guess it makes less sense here.